server/src/main/java/org/moparforia/server/Server.java

```diff
@@ -31,14 +39,16 @@
 public class Server implements Runnable {
 
     public static final boolean DEBUG = true;
+    public static final String DEFAULT_TRACKS_DIRECTORY = "tracks";
 
-    private HashMap<Integer, Player> players = new HashMap<Integer, Player>();
+    private HashMap<Integer, Player> players = new HashMap<>();
     private ChannelGroup allChannels = new DefaultChannelGroup();
-    private ConcurrentLinkedQueue<Event> events = new ConcurrentLinkedQueue<Event>();
-    private HashMap<PacketType, ArrayList<PacketHandler>> packetHandlers = new HashMap<PacketType, ArrayList<PacketHandler>>();
+    private ConcurrentLinkedQueue<Event> events = new ConcurrentLinkedQueue<>();
```

Comment:
why change this?

Reply:
Since I was touching this code anyway, I figured I might as well clean it up. The extra generic parameters on the right hand side of the assignment add no extra information but is just more verbose and IntelliJ was complaining about it:
![image](https://github.com/PhilippvK/playforia-minigolf/assets/21343173/4880d446-3860-4dcc-b6b9-4ec6a050806b)